ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -34,13 +31,23 @@
       </div>
       <div class="is-flex-row">
         {{#if this.mfaAuthData}}
-          <button type="button" class="icon-button" {{on "click" (fn (mut this.mfaAuthData) null)}}>
-            <Icon @name="arrow-left" @size="24" aria-label="Back to login" class="icon-blue" />
-          </button>
+          <Hds::Button
+            @text="Back to login"
+            @icon="arrow-left"
+            @isIconOnly={{true}}
+            @color="tertiary"
+            @size="large"
```

Comment:
Size seems much larger than 24px (it's showing as 48px for me) I think we could probably remove the size arg here and below and these buttons will already get larger when we increase our base font size